### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by Demarchist1**

### CODE REVIEW

File: src/scss/global.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@ body {
   font-family: "Rubik", sans-serif;
   height: 100vh;
   overscroll-behavior-x: none;
+  overflow: hidden;
```
Comment (thecadams) [inline src/scss/global.scss:22]: why are we preventing scrolling in the entire app?

File: src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx [hunk 2]
```diff
@@ -24,12 +24,10 @@ const SparkleFairiesPopUp: React.FunctionComponent<PropsType> = ({
           <p className="title-sidebar">{`If you're looking for information, please check out these resources:`}</p>
           <a
             href={PLAYA_INFO_URL}
-            rel="noopener noreferrer"
-            target="_blank"
             className="btn btn-primary join-button"
             style={{ fontSize: 14 }}
           >
-            Playa Information Booth
+            Paddock Info Booth
```
Comment (thecadams) [inline src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx:30]: nit: can you please move this to settings as well?

File: src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx [hunk 3]
```diff
@@ -44,7 +42,7 @@ const SparkleFairiesPopUp: React.FunctionComponent<PropsType> = ({
             className="btn btn-primary join-button"
             style={{ fontSize: 14 }}
           >
-            Launch Ranger Chat
+            Red Earth Ranger Chat
```
Comment (thecadams) [inline src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx:45]: nit: can you please move this to settings as well?

File: src/components/organisms/InformationLeftColumn/InformationLeftColumn.tsx [hunk 4]
```diff
@@ -49,14 +49,14 @@ const InformationLeftColumn: React.FunctionComponent<PropsType> = ({
               isLeftColumnExpanded ? "expanded-popup" : ""
             }`}
           />
-        ) : venueLogoPath === "heart" && isLeftColumnExpanded ? (
-          <img
-            src="/imageDonate1.png"
-            alt="experience-logo"
-            className={`expanded-popup`}
+        ) : venueLogoPath === "heart" ? (
+          <FontAwesomeIcon
```
Comment (thecadams) [inline src/components/organisms/InformationLeftColumn/InformationLeftColumn.tsx:53]: Would like to understand a little more where the heart went 😂 considering this component is used in many places in the code.

**Review by thecadams — COMMENTED** [review]
Hey some stuff here has me looking for clarification a little overall though glad to see things getting cleaned up.

File: src/components/organisms/InformationLeftColumn/InformationLeftColumn.tsx (hunk 4 above)
Comment (Demarchist1) [reply src/components/organisms/InformationLeftColumn/InformationLeftColumn.tsx:53]: > Would like to understand a little more where the heart went 😂 considering this component is used in many places in the code.

This version is putting the heart back, I removed the burning man project graphic that appeared there before.

Comment (thecadams) [reply src/components/organisms/InformationLeftColumn/InformationLeftColumn.tsx:53]: Aha thanks!

**Review by thecadams — APPROVED** [review]
Great stuff! Thanks for this!

**Pull request merged**